src/stcal/jump/jump.py

```diff
@@ -246,16 +248,17 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
         n_slices = max_available
 
     if n_slices == 1:
-        gdq, row_below_dq, row_above_dq = \
+        gdq, row_below_dq, row_above_dq, total_primary_crs, stddev = \
             twopt.find_crs(data, gdq, readnoise_2d, rejection_thresh,
                            three_grp_thresh, four_grp_thresh, frames_per_group,
                            flag_4_neighbors, max_jump_to_flag_neighbors,
                            min_jump_to_flag_neighbors, dqflags,
                            after_jump_flag_e1=after_jump_flag_e1,
                            after_jump_flag_n1=after_jump_flag_n1,
                            after_jump_flag_e2=after_jump_flag_e2,
-                           after_jump_flag_n2=after_jump_flag_n2)
-
+                           after_jump_flag_n2=after_jump_flag_n2, copy_arrs=False,
+                           minimum_groups=3, minimum_sigclip_groups=minimum_sigclip_groups,
```

Comment:
Do you really want to hard code the value passed for `minimum_groups` or should it instead use the param value passed into the function (as is done here with `minimum_sigclip_groups`)?

Reply:
Well, the old version of this used to have a hard coded lower limit. I put in the parameter of a lower limit in the STCAL version but I just don't see the need to expose this to JWST version until we come up with a reason.